aspnet/security/introduction-to-aspnet-identity.rst

```diff
@@ -0,0 +1,100 @@
+
+	.. image:: introduction-to-aspnet-identity/_static/02-reg.png
+
+When the user clicks the **Register** button, the ``Register`` action of the ``AccountController`` creates the user by calling ``CreateAsync`` function of the ``UserManager`` object, as shown below:
+
+.. literalinclude:: introduction-to-aspnet-identity/sample/src/ASPNET-IdentityDemo/Controllers/AccountController.cs
+	:language: c#
+	:lines: 101-127
+	:emphasize-lines: 4,10
+	:dedent: 8
+	
+4. Log in.
+ If the user was successfully created, they are logged in by the ``SignInAsync`` method, also contained in the ``Register`` action.
+	
+.. literalinclude:: introduction-to-aspnet-identity/sample/src/ASPNET-IdentityDemo/Controllers/AccountController.cs
+	:language: c#
+	:lines: 101-127
+	:emphasize-lines: 19
+	:dedent: 8
+
+The above ``SignInAsync`` method calls the below ``SignInAsync`` task, which is contained in the ``SignInManager`` class. The ``SignInAsync`` method is overloaded. The ``authenticationProperties`` parameter that can be added to the call. This parameter is a dictionary used to store state values about the authentication session. These values are applied to the login and authentication cookie. 
+
+5. Log off.
+ Clicking the **Log off** link calls the ``LogOff`` action in the account controller. 
+ 
+.. literalinclude:: introduction-to-aspnet-identity/sample/src/ASPNET-IdentityDemo/Controllers/AccountController.cs
+	:language: c#
+	:lines: 131-138
+	:emphasize-lines: 5
+	:dedent: 8 
+ 
+The code above shows the ``SignInManager.SignOut`` method. This is analogous to `FormsAuthentication.SignOut <http://msdn.microsoft.com/en-us/library/system.web.security.formsauthentication.signout.aspx>`_ method used by the `FormsAuthentication <http://msdn.microsoft.com/en-us/library/system.web.security.formsauthenticationmodule.aspx>`_ module in Web Forms.
+
+6. View the database.
+ After stopping the application, view the user database from Visual Studio by selecting **View** -> **SQL Server Object Explorer**. Then, expand the following within the **SQL Server Object Explorer**:
+ 
+ - (localdb)\MSSQLLocalDB
+ - Databases
+ - aspnet5-<*the name of your application*>
+ - Tables
+
+ Next, right-click the **dbo.AspNetUsers** table and select **View Data** to see the properties of the user you created.
+
+	.. image:: introduction-to-aspnet-identity/_static/04-db.png
+
+Components of ASP.NET Identity
+------------------------------
+
+The diagram below shows the components of the ASP.NET Identity system (click on `this <http://i1.asp.net/media/4459023/1.png?cdn_id=2015-08-15-002>`_ or on the diagram to enlarge it). The packages in green make up the ASP.NET Identity system. All the other packages are dependencies which are needed to use the ASP.NET Identity system in ASP.NET applications.  
```

Comment:
Is this diagram up-to-date? I don't think the dnx packages use the "Owin" names, and EF is EF7. At a higher level, is this diagram useful? I can see the dependency graph in VS.